### DISCUSSION TITLE
NOCODB 应用数据库参数配置错误

### DISCUSSION THREAD

**Discussion by hhhguany** [1Panel-dev/1Panel#6346] [Q&A]
### 1Panel 版本

v1.10.14-lts

### 问题描述

NOCODB 更新后启动失败，查看日志报错 “Cannot find module 'mysql'”。

根据检索，查询到 [解决方案](https://github.com/nocodb/nocodb/issues/164)，需要将数据库类型从 `mysql` 改为 `mysql2`

### 重现步骤

- 1panel 中安装 mysql 数据库
- 安装 NOCODB 应用，升级后，无法打开

**Comment by wanghe-fit2cloud** [comment]
> 刚在本地环境未能复现上述问题

通过应用商店安装 NocoDB 时，系统会执行初始化脚本来修改数据库类型字段，如图：
![image](https://github.com/user-attachments/assets/3cd936d1-cf79-4285-93e4-69976628e42c)


安装成功后，`.env` 环境变量文件中的数据库类型字段已被修改为 `mysql2`，服务可以正常访问，如图：
![image](https://github.com/user-attachments/assets/542d407a-c8ac-4a3d-98a0-ade0baf777c7)

**Comment by r4ym0n** [comment]
这里遇到了同样的问题，修改为 mysql2 之后恢复正常
